app/domain/errors.rb

```diff
@@ -0,0 +1,213 @@
+# frozen_string_literal: true
+
+require 'util/error_class'
+
+unless defined? Errors::Authentication::AuthenticatorNotFound
+  # this wrapper is here so these classes will not be loaded by Rails
+  # auto-load. #TODO: fix this in a proper manner
```

Comment:
What does "Fix this in a proper manner" mean - is that a comment for me?

Reply:
this is actually a note for future developers :)